cob_bringup/drivers/ur.launch

```diff
@@ -33,6 +34,7 @@
 	<include if="$(arg sim)" file="$(find cob_bringup)/controllers/generic_controller.launch">
 		<arg name="robot" value="$(arg robot)"/>
 		<arg name="component_name" value="$(arg component_name)"/>
+		<arg name="pkg_hardware_config" value="$(arg pkg_hardware_config)"/>
```

Comment:
what do you need this for?
why can't you adjust the yaml's in `cob_hardware_config`?

Reply:
If I want to launch a configuration from a specific project. That is why all the other files in the chain have it, isn't it?